content/hi/training/_index.html

```diff
@@ -0,0 +1,154 @@
+        <div class="logo-certification cta-image" id="logo-kcnf">
+          <img src="/images/training/kubernetes-kcnf-white.svg" />
+        </div>
+        <div class="logo-certification cta-image" id="logo-kcsa">
+          <img src="/images/training/kubernetes-cksa-white.svg" />
+        </div>
+        <div class="logo-certification cta-image" id="logo-cka">
+          <img src="/images/training/kubernetes-cka-white.svg"/>
+        </div>
+        <div class="logo-certification cta-image" id="logo-ckad">
+          <img src="/images/training/kubernetes-ckad-white.svg"/>
+        </div>
+        <div class="logo-certification cta-image" id="logo-cks">
+          <img src="/images/training/kubernetes-cks-white.svg"/>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section>
+  <div class="main-section padded">
+    <center>
+      <h2>edX पर निःशुल्क पाठ्यक्रम लें</h2>
+    </center>
+  <div class="col-container">
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>कुबेरनेट्स का परिचय <br> &nbsp;</b>
+        </h5>
+        <p>कुबेरनेट्स सीखना चाहते हैं? कंटेनरीकृत अनुप्रयोगों के प्रबंधन के लिए इस शक्तिशाली प्रणाली पर गहन प्राइमर प्राप्त करें।</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-kubernetes" target="_blank" class="button" hreflang="en">कोर्स पर जाएँ</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>क्लाउड इंफ्रास्ट्रक्चर टेक्नोलॉजीज का परिचय</b>
+        </h5>
+        <p>ओपन सोर्स में अग्रणी, लिनक्स फाउंडेशन से सीधे क्लाउड प्रौद्योगिकियों के निर्माण और प्रबंधन की बुनियादी बातें सीखें।</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-cloud-infrastructure-technologies" target="_blank" class="button"  hreflang="en">कोर्स पर जाएँ</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>लिनक्स का परिचय</b>
+        </h5>
+        <p>कभी लिनक्स नहीं सीखा? ताज़ा करना चाहते हैं? प्रमुख लिनक्स वितरण परिवारों में ग्राफिकल इंटरफ़ेस और कमांड लाइन दोनों का उपयोग करके लिनक्स का अच्छा कार्यसाधक ज्ञान विकसित करें।</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-linux" target="_blank" class="button" hreflang="en">कोर्स पर जाएँ</a>
+      </center>
+    </div>
+  </div>
+</section>
+
+<div class="padded lighter-gray-bg">
+  <div class="main-section two-thirds-centered">
+    <center>
+      <h2>लिनक्स फाउंडेशन के साथ सीखें</h2>
+      <p>लिनक्स फाउंडेशन कुबेरनेट्स अनुप्रयोग विकास और संचालन जीवनचक्र के सभी पहलुओं के लिए प्रशिक्षक के नेतृत्व वाले और स्व-गति वाले पाठ्यक्रम प्रदान करता है।</p>
+      <br/><br/>
+      <a href="https://training.linuxfoundation.org/training/course-catalog/?_sft_technology=kubernetes" target="_blank" class="button">पाठ्यक्रम देखें</a>
+    </center>
+  </div>
+</div>
+
+<section id="get-certified">
+  <div class="main-section padded">
+    <h2>कुबेरनेट्स प्रमाणित प्राप्त करें</h2>
```

Comment:
While in English, it is okay to say Get Kubernetes Certified, in Hindi, it doesn't translate well. Kindly check machine translations to ensure whatever is written can actually be spoken and understood by a layperson.

```suggestion
    <h2>कुबेरनेट्स प्रमाणित बनें</h2>
```